recipe/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set name = "openvino" %}
-{% set version_postfix = "2024.2.0" %}
-{% set version = "2024.2.0.dev20240513" %}
+{% set version_postfix = "2024.3.0" %}
+{% set version = "2024.3.0.dev20240621" %}
 
 package:
   name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
-  - url: https://github.com/openvinotoolkit/openvino/archive/refs/tags/{{ version }}.tar.gz
-    sha256: a972ed6525889f63e0de1f2484dd8e4c5ed649a64da454caf36986bc40c80a44
+  - url: https://github.com/Wovchena/openvino/archive/4235e7523052ba34a5bd7135b9fd419643021756.tar.gz
+    sha256: b5cb38553d4bc848823577ec6a95a4ec959a92202241da41f007e42a1be20db4
```

Comment:
Why non official repo is used?

Reply:
It doesn't work: https://github.com/conda-forge/openvino-feedstock/pull/100/commits/2831159dd5124b18f07dd39fa22ba54b82f0d628

Reply:
ok, let's fix in original place first